### PR TITLE
feat: add documentation for phone calls [sc-16182]

### PR DESCRIPTION
**Pull request by maxigimenez**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [x] Code is linted (`npm run lint`)

## Notes for the Reviewer
Add documentation for phone calls alerts

### CODE REVIEW

File: site/content/docs/alerting/phone-calls.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    parent: "Alerting"
+---
+
+The phone numbers used for phone calls alerting need to be in E.164 format. Stick to the following rules and you'll be fine:
+
+- Prepend international access codes with a + sign
+- Do not use any white spaces
+- Use up to 15 characters
+
+![send monitoring alerts with SMS](/docs/images/alerting/phone-call.png)
```
Comment (drakirnosslin) [inline site/content/docs/alerting/phone-calls.md:15]: send monitoring alerts with SMS -> send monitoring alerts with phone calls

File: site/content/docs/alerting/phone-calls.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+| India |
+| Norway |
+| Brazil |
+| Netherlands |
+| Vietnam |
+| Canada |
+| Australia |
+
+{{< warning >}}
+If you country is not supported feel free to reach out our [support team](mailto:support@checklyhq.com).
```
Comment (drakirnosslin) [inline site/content/docs/alerting/phone-calls.md:36]: 'your'

File: site/content/docs/alerting/phone-calls.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+{{< /warning >}}
+
+## Phone call number and sender ID
+
+Phone calls alerts will be sent with a sender ID and phone number. But not all carriers support that. For instance, the number the call is sent from in the US will be +18142508623. But in other countries only the sender ID will be displayed, which is "Checkly".
```
Comment (drakirnosslin) [inline site/content/docs/alerting/phone-calls.md:41]: Is the above correct? As discussed on Slack, it seems that it'll always be the number that is displayed.

File: site/content/docs/alerting/phone-calls.md (hunk 2 above)
Comment (drakirnosslin) [reply site/content/docs/alerting/phone-calls.md:36]: Let's use alphabetical order:
+ | Australia |
+ | Brazil |
+ | Canada |
+ | France |
+ | Germany |
+ | India |
+ | Netherlands |
+ | Norway |
+ | Vietnam |
+ | United Kingdom |
+ | United States |

**Review by tnolet — APPROVED** [review]
LGTM

**Review by miliberlin — APPROVED** [review]

**Pull request merged**